### PR TITLE
M1: CRUD todo list

### PR DESCRIPTION
**Pull request by Lucas-Fernandes-Nogueira**

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -11,7 +11,31 @@ <h1 class="display-4">To-Do App</h1>
...
+                  <i class="far fa-square"></i>
+                {% endif %}
+              </button>
+              {{ item.title }}
+            </form>
+          </li>         
+        {% endfor %}
+        <form action="/new-item" method="POST">
+          <button type="submit"  style="background-color:rgba(0, 0, 0, 0);"><i class="fas fa-plus"></i></button>
+          <input type="text" id="title" name="title">
```
Comment (Jonesey13) [inline templates/index.html]: Probably want to add a `required` attribute here

File: templates/index.html [hunk 2]
```diff
@@ -11,7 +11,31 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for item in items %}
+          <li style="list-style-type: none;">
+            <form action="/remove-item" method="POST" style="display: inline;">
```
Comment (Jonesey13) [inline templates/index.html]: If you wanted to you could use "url_for()" here (but not important)

File: templates/index.html [hunk 3]
```diff
@@ -11,7 +11,31 @@ <h1 class="display-4">To-Do App</h1>
...
+              <input type="hidden" id="id" name="id" value={{item.id}} />
+              <button type="submit" style="border: none; background-color:rgba(0, 0, 0, 0);">
+                <i class="fas fa-trash-alt" style="color:red;"></i>
+              </button>
+            </form>
+            <form action="/toggle-status" method="POST" style="display: inline;">
+              <input type="hidden" id="id" name="id" value={{item.id}} />
+              <button type="submit" style="border: none; background-color:rgba(0, 0, 0, 0);">
+                {% if item.status == 'Completed' %} 
+                  <i class="far fa-check-square"></i>
```
Comment (Jonesey13) [inline templates/index.html:26]: Nice styling with the icons!

File: app.py [hunk 4]
```diff
@@ -4,9 +4,38 @@
...
+    item = session.get_item(itemId)
+
+    if item["status"] == 'Not Started':
+        item["status"] = 'Completed'
+    else:
+        item["status"] = 'Not Started'
+
+    session.save_item(item)
+
+    return redirect(url_for('index'))
```
Comment (Jonesey13) [inline app.py:32]: Good catch for noticing the need to use a redirect

File: app.py [hunk 5]
```diff
@@ -4,9 +4,38 @@
...
+
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = session.get_items()
+    return render_template('index.html', items=items)
+
+@app.route('/new-item', methods=['POST'])
+def addItem():
+    title = request.form.get('title')
```
Comment (Jonesey13) [inline app.py:15]: You could add some server side validation here

**Review by Jonesey13 — APPROVED** [review]
Looks Great! All functionally is sound and there's only really one minor comment about validation 

Feel free to merge in this PR whenever you are ready :)

File: app.py (hunk 5 above)
Comment (Lucas-Fernandes-Nogueira) [reply app.py:15]: Done 😄

File: templates/index.html (hunk 2 above)
Comment (Lucas-Fernandes-Nogueira) [reply templates/index.html]: Done it!

File: templates/index.html (hunk 1 above)
Comment (Lucas-Fernandes-Nogueira) [reply templates/index.html]: Good point

**Pull request merged**